libnavui-shield/src/main/java/com/mapbox/navigation/ui/shield/ShieldsCache.kt

```diff
@@ -195,7 +196,7 @@ internal class ShieldResultCache(
     }
 }
 
-internal class ShieldSpritesCache : ResourceCache<String, ShieldSprites>() {
+internal class ShieldSpritesCache : ResourceCache<String, ShieldSprites>(3) {
```

Comment:
This however should only be downloaded once per style, I'd imagine a dozen or so should be enough.

Reply:
I would not go for even dozen. If we think about it there are probably 3 styles:

- Street: Day/Night
- Terrain: Day/Night
- Satellite: Day/Night

I think 8 should be a good number in case someone adds another kind with both day/night variants.

Reply:
Any reason why this size isn't updated?
@cafesilencio

Reply:
I was waiting for the dialog between you and Lukas to conclude and then moved on to other issues.  I can make it 8 if you think that's the right number.  It's easy enough to adjust later.